lib/makeWebpackConfig.js

```diff
@@ -180,7 +180,15 @@ module.exports = async (playroomConfig, options) => {
         filename: 'preview/index.html',
         publicPath: '../',
       }),
-      new VanillaExtractPlugin(),
+      new VanillaExtractPlugin({
+        test: (filePath) => {
+          // Only apply VanillaExtract plugin to playroom's source `.css.ts` files
+          return (
+            /\.css\.ts$/i.test(filePath) &&
+            includePaths.some((playRoomPath) => filePath.includes(playRoomPath))
```

Comment:
`includePaths` are absolute, so `startsWith` should work here and be slightly faster than `includes`:
```suggestion
            /\.css\.ts$/i.test(filePath) &&
            includePaths.some((playroomPath) =>
              filePath.startsWith(playroomPath)
            )
```